src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/ClaimStore.java

```diff
@@ -57,9 +57,11 @@ public Claim saveClaim(ClaimData claimData, String submitterId, LocalDate respon
             SampleClaim.SUBMITTER_EMAIL
         );
 
-        logger.info("Saved claim has been given ID " + claimId + ". The following claims exist in the DB: "
-            + this.claimRepository.findAll().stream().map(claim -> claim.getId() + " - " + claim.getExternalId())
-            .collect(Collectors.toList()));
+        logger.debug("Saved claim has been given ID {}. The following claims exist in the DB: {}",
+            claimId,
+            this.claimRepository.findAll().stream().map(claim -> claim.getId() + " - " + claim.getExternalId())
```

Comment:
Please don't execute an extra database call in order to log something out on debug, this has performance impact!